src/clp_ffi_py/ErrorMessage.hpp

```diff
@@ -0,0 +1,12 @@
+#ifndef CLP_FFI_PY_ERROR_MESSAGE
+#define CLP_FFI_PY_ERROR_MESSAGE
+
+namespace clp_ffi_py::error_messages {
+namespace encoder {
+    constexpr char const* cTimestampError =
```

Comment:
Why `constexpr char const*` instead of `constexpr char var[]`? If it's because of some clang-tidy thing, you should add a comment so you don't get this question again in the future.